rest_framework-stubs/serializers.pyi

```diff
@@ -1,16 +1,14 @@
 from collections.abc import Callable, Iterable, Iterator, Mapping, MutableMapping, Sequence
 from typing import Any, Generic, NoReturn, TypeVar
-from _typeshed import Self
 
+from _typeshed import Self
 from django.core.exceptions import ValidationError as DjangoValidationError
 from django.db import models
 from django.db.models import DurationField as ModelDurationField
 from django.db.models import Manager, Model, QuerySet
 from django.db.models.fields import Field as DjangoModelField
-from django.utils.translation import ugettext_lazy as _
 from django.utils.functional import cached_property
-from typing_extensions import Literal
-
+from django.utils.translation import ugettext_lazy as _
```

Comment:
isort also made changes to the import order, I thought lint was checking that already?